inc/cachify.class.php

```diff
@@ -651,7 +651,7 @@ public static function add_dashboard_count( $items = array() )
 			esc_attr(strtolower($method)),
 			esc_html__( 'Caching method', 'cachify' ),
 			esc_attr($method),
-			( empty($size) ? esc_html__( 'Empty', 'cachify' ) : size_format($size) )
+			( empty($size) ? esc_html__( 'Empty Cache', 'cachify' ) : size_format($size) . ' ' . esc_html__( 'Cache', 'cachify' ) )
```

Comment:
@timse201 I didn't mean the "Empty Cache" line. That's fine. Changing that would be even worse. What I mean is something along these lines:

```php
if ( empty( $size ) ) {
  esc_html__( 'Empty Cache', 'cachify' )
} else {
  /* translators: %s: cache size */
  sprintf( esc_html__( '%s Cache', 'cachify' ), size_format($size) );
}
```

Why? Because `size_format($size) . ' ' . esc_html__( 'Cache', 'cachify' ) )` is not translator friendly. Think about right-to-left languages etc.